scipy/_lib/_util.py

```diff
@@ -458,6 +458,51 @@ def __call__(self, func, iterable):
                             " form f(func, iterable)") from e
 
 
+def rng_random(gen, *, size=None, dtype=np.float64, out=None):
```

Comment:
I would not do this. We will soon drop support for numpy <1.17 and we should not really add more hack around it. Right now you can just use `rng.uniform` for this.

Reply:
OK. I reverted it.

Reply:
I have seen thanks.